central/config/datastore/singleton.go

```diff
@@ -41,11 +41,11 @@ const (
 	// PlatformComponentSystemRuleName is the name of the system defined rule for matching openshift and kube workloads
 	PlatformComponentSystemRuleName = "system rule"
 	// PlatformComponentSystemRegex is the system defined regex for matching kube and openshift workloads
-	PlatformComponentSystemRegex = `^kube-.*|^openshift-.*`
+	PlatformComponentSystemRegex = `^openshift$|^openshift-apiserver$|^openshift-operators$|^kube-.*`
 	// PlatformComponentLayeredProductsRuleName is the name of the system defined rule for matching workloads created by Red hat layered products
 	PlatformComponentLayeredProductsRuleName = "red hat layered products"
 	// PlatformComponentLayeredProductsDefaultRegex is the default regex for matching workloads created by Red hat layered products
```

Comment:
I think this comment and the naming in general is in-accurate. Based on your comments in this PR, it seems that the main distinction is not between system and layered product, but between which rules are static vs editable. If you want to stick with the system/layered product naming, I would at least put a comment here explaining the additional meaning.